src/container/cli_tx.c

```diff
@@ -1,456 +0,0 @@
-/**
```

Comment:
I think all transaction operations are per container and not per object, so i don't think it's right moving those to src/object.

Reply:
The DTX operated targets are all object based, not container based, so I think it should be inside src/object/.

The 2nd: DTX client logic will call object related functions, if we move it into src/container/, then it will depends on the exported object functions. On the other hand, some functions inside src/object/ will also need to call DTX related functions, then will depend on the exported functions from src/container/. So it will be more convenient if the DTX logic is inside src/object/ to remove interdependency.

Reply:
yea i understand transaction operations are object based, but you could space multiple object per tx, and a tx is at a container scope.
but given that it introduces complexity to keep it in container, then im fine if you want to move it.